examples/ts-vue2/README.md

```diff
@@ -4,10 +4,10 @@ Demo app to showcase usage of the vue web component in a ts class-style vue app.
 
 ## Project setup
 ```
-npm install
+yarn install
 ```
 
 ### Run demo
 ```
-npm run serve
+yarn serve
```

Comment:
it's working well, however, there are some type issues

Reply:
Not a priority to fix this atm.